lib/frozen_record/test_helper.rb

```diff
@@ -0,0 +1,41 @@
+module FrozenRecord
+  module TestHelper
+    NoFixturesLoaded = Class.new(StandardError)
+
+    def self.load_fixture(model_class, alternate_base_path)
```

Comment:
Please use `class << self` instead of `def self.` to be consistent with the rest of the codebase.